docs/BuildOptions.md

```diff
@@ -174,275 +153,236 @@ as the `inet_pton` function gets a poor-man's simple replacement that can
+that it may cause problems in case of default installation.
 
-As both are ON by default, this option allows you for turning
-off building a library in a style that is not of your interest,
-for example, leave only static one by `--disable-shared`. You
-can't disable both at once.
 
+**`--enable-shared`** and **`--enable-static`** (default for both: ON)
 
-`--enable-testing` (default: OFF)
----------------------------------
+Enables building SRT as a shared and/or static library, as required for your 
+application. The only practical use is by disabling one of them
+(e.g. by `--disable-shared`). Note that you can't disable both at once.
 
-Enables compiling developer testing appliactions.
 
+**`--enable-testing`** (default: OFF)
 
-`--enable-thread-check` (default: OFF)
---------------------------------------
+Enables compiling of developer testing applications.
 
-Enable `#include <threadcheck.h>` that implements `THREAD_*` macros".
 
-This is a solution used by one of the users to support better thread
-debugging.
+**`--enable-thread-check`** (default: OFF)
 
+Enables `#include <threadcheck.h>`, which implements `THREAD_*` macros" to 
+support better thread debugging. This is used by one of dependent projects.
 
-`--enable-unittests` (default: OFF)
------------------------------------
 
-Developer option. When ON, the unit tests, possibly with downloading and installing
-the google test library in the build directory, will be enabled, and the tests will
-be run as part of the build process.
+**`--enable-unittests`** (default: OFF)
 
+When ON, this option enables unit tests, possibly with the download 
+and installation of the Google test library in the build directory. The tests 
+will be run as part of the build process. This is predicted for developers only.
 
-`--openssl-crypto-library=<filepath>`
--------------------------------
+
+**`--openssl-crypto-library=<filepath>`**
 
 Configure the path to an OpenSSL Crypto library.
 
 
-`--openssl-include-dir=<path>`
--------------------------------
+**`--openssl-include-dir=<path>`**
 
-Configure the path to include files for OpenSSL library.
+Configure the path to include files for an OpenSSL library.
 
 
-`--openssl-ssl-library=<filepath>`
--------------------------------
+**`--openssl-ssl-library=<filepath>`**
 
 Configure the path to an OpenSSL SSL library.
 
 
-`--pkg-config-executable=<filepath>`
--------------------------------
+**`--pkg-config-executable=<filepath>`**
 
-Configure the path to `pkg-config` tool.
+Configure the path to the `pkg-config` tool.
 
 
-`--prefix=<path>`
------------------
+**`--prefix=<path>`**
 
-This is an alias to `--cmake-install-prefix`. This is the root
-directory for installation, inside which next GNU/POSIX compatible
-directory layout will be used for particular parts. As on all
-known build systems, this defaults to `/usr/local` on POSIX
-compatible systems.
+This is an alias to `--cmake-install-prefix`. This is the root directory for
```

Comment:
This is an alias to the `--cmake-install-prefix` variable that establishes the 
root directory for installation, inside of which a GNU/POSIX compatible directory layout will be used. As on all known build systems, this defaults to `/usr/local` 
on GNU/POSIX compatible systems, with lower level GNU/POSIX directories 
created inside: `/usr/local/bin`,`/usr/local/lib`, etc.